src/Settings/AnonymizedDataConsentScreen.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { FunctionComponent } from "react"
+import {
+  Text,
+  View,
+  StyleSheet,
+  ScrollView,
+  TouchableOpacity,
+} from "react-native"
+import { useNavigation } from "@react-navigation/native"
+import { useTranslation } from "react-i18next"
+import { SvgXml } from "react-native-svg"
+
+import { useConfigurationContext } from "../ConfigurationContext"
+import { useAnalyticsContext } from "../AnalyticsContext"
+import { useStatusBarEffect, Stacks } from "../navigation"
+
+import { Button } from "../components"
+import { Colors, Typography, Spacing, Iconography } from "../styles"
+import { Icons } from "../assets"
+import ExternalLink from "./ExternalLink"
+
+const AnonymizedDataConsentScreen: FunctionComponent = () => {
+  useStatusBarEffect("dark-content", Colors.primaryLightBackground)
+  const { t } = useTranslation()
+  const navigation = useNavigation()
+  const {
+    healthAuthorityName,
+    healthAuthorityPrivacyPolicyUrl,
+  } = useConfigurationContext()
+  const { updateUserConsent } = useAnalyticsContext()
+
+  const handleOnPressConsent = async () => {
+    await updateUserConsent(true)
+    navigation.navigate(Stacks.Settings)
+  }
+
+  const handleOnPressClose = () => {
+    navigation.navigate(Stacks.Settings)
+  }
+
+  return (
```

Comment:
I think we need a `<StatusBar backgroundColor={Colors.primaryLightBackground} />` here.